examples/client/client.c

```diff
@@ -1099,455 +1099,674 @@ static int ClientRead(WOLFSSL* ssl, char* reply, int replyLen, int mustRead,
+            "            フォルダー中のファイルは、\"hash.N\"[N:0-9]名である必要があります\n"
+            "            以下の例ではca-cert.pemにシンボリックリンクを設定します\n"
+            "            ln -s ca-cert.pem  `openssl x509 -in ca-cert.pem -hash -noout`.0\n",
+    },
 #endif
 #if defined(WOLFSSL_WOLFSENTRY_HOOKS) && !defined(NO_FILESYSTEM) && \
     !defined(WOLFSENTRY_NO_JSON)
-        "--wolfsentry-config <file>    Path for JSON wolfSentry config\n",
-                                                                        /* 68 */
+        {
+            OPTION_TYPE_WOLFSENTRY_CONFIG_FILE, '--wolfsentry-config',
+            "--wolfsentry-config <file>    Path for JSON wolfSentry config\n",
+            "--wolfsentry-config <file>    wolfSentry コンフィグファイル\n",
+    },
 #endif
 #ifndef WOLFSSL_TLS13
-        "-7          Set minimum downgrade protocol version [0-3] "
-        " SSLv3(0) - TLS1.2(3)\n",
+        {
+            OPTION_TYPE_SET_MIN_DOWNGRADE_PROTOCOL_VER, '7',
+            "-7          Set minimum downgrade protocol version [0-3] "
+            " SSLv3(0) - TLS1.2(3)\n",
+            "-7          最小ダウングレード可能なプロトコルバージョンを設定します [0-3] "
+            " SSLv3(0) - TLS1.2(3)\n",
+
+    },
 #else
-        "-7          Set minimum downgrade protocol version [0-4] "
-           " SSLv3(0) - TLS1.3(4)\n",                                   /* 69 */
+        {
+            OPTION_TYPE_SET_MIN_DOWNGRADE_PROTOCOL_VER, '7',
+            "-7          Set minimum downgrade protocol version [0-4] "
+           " SSLv3(0) - TLS1.3(4)\n",
+            "-7          最小ダウングレード可能なプロトコルバージョンを設定します [0-4] "
+            " SSLv3(0) - TLS1.3(4)\n",
+    },
 #endif
 #ifdef HAVE_PQC
-        "--pqc <alg> Key Share with specified post-quantum algorithm only [KYBER_LEVEL1, KYBER_LEVEL3,\n"
+        {
+            OPTION_TYPE_PQC, '--pqc',
+            "--pqc <alg> Key Share with specified post-quantum algorithm only [KYBER_LEVEL1, KYBER_LEVEL3,\n"
             "            KYBER_LEVEL5, KYBER_90S_LEVEL1, KYBER_90S_LEVEL3, KYBER_90S_LEVEL5,\n"
             "            NTRU_HPS_LEVEL1, NTRU_HPS_LEVEL3, NTRU_HPS_LEVEL5, NTRU_HRSS_LEVEL3,\n"
             "            SABER_LEVEL1, SABER_LEVEL3, SABER_LEVEL5, P256_NTRU_HPS_LEVEL1,\n"
             "            P384_NTRU_HPS_LEVEL3, P521_NTRU_HPS_LEVEL5, P384_NTRU_HRSS_LEVEL3,\n"
             "            P256_SABER_LEVEL1, P384_SABER_LEVEL3, P521_SABER_LEVEL5, P256_KYBER_LEVEL1,\n"
             "            P384_KYBER_LEVEL3, P521_KYBER_LEVEL5, P256_KYBER_90S_LEVEL1, P384_KYBER_90S_LEVEL3,\n"
-            "            P521_KYBER_90S_LEVEL5]\n",                         /* 70 */
-#endif
-#ifdef WOLFSSL_SRTP
-        "--srtp <profile> (default is SRTP_AES128_CM_SHA1_80)\n",       /* 71 */
-#endif
-        "\n"
-           "For simpler wolfSSL TLS client examples, visit\n"
-           "https://github.com/wolfSSL/wolfssl-examples/tree/master/tls\n", /* 72 */
-        NULL,
-    },
-#ifndef NO_MULTIBYTE_PRINT
```

Comment:
Please consider to be available #define NO_MULTIBYTE_PRINT use on this re-organization.

When NO_MULTIBYTE_PRINT is defined, we can omit Japanese messages so that total code size becomes smaller than including those messages. Also, there could be environment that doesn't support multi byte. In that case, we should give option to compile out Japanese messages.

Reply:
I understand. Since NO_MULTIBYTE_PRINT was not defined in the program, I did not think it was necessary. But I should have considered some environment couldn't support multi byte.

Reply:
Thanks. When you grep `NO_MULTIBYTE_PRINT` to files in wolfSSL repo, you can find multiple places to use the define.

Reply:
I think that your re-organization is nice because we can see English and Japanese message at once. That helps to translate message much. But, to disable Japanese message, we might need to surround #ifndef-#endif each line. That is annoying :-<

Therefore, we could introduce a macro, something likes as follows:
```
#define OPTMSG(t, s, eng, jp) \
  o, s, eng, \
  #ifndef NO_MULTIBYTE_PRINT \
    jpn
  #else
    NULL
  #endif

const struct OptionsMap options[] = { 
        {  
           OPTMSG(
            MSG_FIRST_NOTE, '*',  
            " NOTE: All files relative to wolfSSL home dir\n",
            " 注意 : 全てのファイルは wolfSSL ホーム・ディレクトリからの相対です。\n"
            )
    },
```
Please adjust macro as you need. Hope help you.

Reply:
Thank you very much.
This approach is very helpful.
I thought that there are only way to disable Japanese message by surrounding with #ifndef-#endif and this is really annoying.
But, the approach you showed will make it a more readable.